src/main/java/com/ays/assignment/model/dto/response/AssignmentsResponse.java

```diff
@@ -0,0 +1,38 @@
+package com.ays.assignment.model.dto.response;
+
+import com.ays.assignment.model.enums.AssignmentStatus;
+import com.ays.common.model.dto.response.BaseResponse;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder
+public class AssignmentsResponse extends BaseResponse {
+
+    private String id;
+    private AssignmentStatus status;
+    private String description;
+    private String firstName;
+    private String lastName;
+    private Location location;
+    private User user;
+
+    @Getter
+    @Setter
+    public static class Location {
+        private Double longitude;
+        private Double latitude;
+    }
+
+    @Getter
+    @Setter
+    public static class User {
```

Comment:
Mapstruct ile birlikte bir problem yaşanmıyorsa Inner class'lar dışarıdan çağrılamasın diye `private` yapmamız daha doğru olabilir

Reply:
MapStruct erişemediği için hata fırlatıyor. @agitrubard